ui/ui_test.go

```diff
@@ -0,0 +1,33 @@
+package pitchforkui
+
+func ExamplePfUI_Page_show() {
```

Comment:
go vet

ui/ui_test.go:3: ExamplePfUI_Page_show refers to unknown field or method: PfUI.Page